components/brave_wallet/common/brave_wallet.mojom

```diff
@@ -1335,6 +1328,34 @@ interface JsonRpcService {
                       string chain_id) => (string decimals,
                                            ProviderError error,
                                            string error_message);
+
+  // Fetches name of an ERC20 or ERC721 token.
+  GetEthTokenName(string contract_address,
```

Comment:
Is it possible not to expose GetEthTokenName and GetEthTokenDecimals + GetEthTokenSymbol above in mojom, but reuse GetEthTokenInfo for
https://github.com/brave/brave-core/blob/c65fcbaf0f5d2f8527e75a76cbfc3bfd06aea020/components/brave_wallet_ui/components/extension/swap/index.tsx#L141-L181

Reply:
Fixed here: 17a0e1f

Also cc: @StephenHeaps, since he may be using those removed methods.